parser/v1/parser.go

```diff
@@ -358,33 +355,15 @@ func (p *Parser) feesTransactions(msg *typesV1.InvocResultV1, tipset *types.Exte
 			BurnAddress: parser.BurnAddress,
 			Amount:      msg.GasCost.BaseFeeBurn.String(),
 		},
+		Amount: msg.GasCost.TotalCost.Int,
```

Comment:
This should be string here... as it will overflow when JSON parsed.

Reply:
Maybe something like TotalCost, instead of just Amount? Something more meaningful